### PR TITLE
Features spring#3 // Lesson#12

### PR DESCRIPTION
**Pull request by yeargor**

### CODE REVIEW

File: src/main/java/Homework11/controller/HelloController.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+@RestController
+@RequiredArgsConstructor
+public class HelloController {
+
+    private final TicketRepository ticketRepository;
+
+    @GetMapping("/protected")
+    public Optional<Ticket> protectedEndpoint(){
+        return ticketRepository.findById(5);
```
Comment (Alexmansar) [inline src/main/java/Homework11/controller/HelloController.java:19]: You need to create some service that will work with repo. In this service you need to prepare data for the controller. In your case - your service should return a Ticket. Not Optional. This conversion from Optional to Ticket is the responsibility of the service.

File: src/main/java/Homework11/model/User.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import lombok.experimental.FieldDefaults;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+import java.time.LocalDate;
+import java.util.Set;
+
+@Getter
+@Setter
+@Entity
+@EntityListeners(AuditingEntityListener.class)
```
Comment (Alexmansar) [inline src/main/java/Homework11/model/User.java:15]: why we need it?

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@yeargor Hi) I have small comments. Please check and fix it)

File: src/main/java/Homework11/model/User.java (hunk 2 above)
Comment (yeargor) [reply src/main/java/Homework11/model/User.java:15]: @Alexmansar, If we are talking about @EntityListeners(AuditingEntityListener.class), so it's needed to automatically generate date on "creationDate" field, which is also marked with @CreatedDate annotation. Those two annotations working together as much as I know

File: src/main/java/Homework11/controller/HelloController.java (hunk 1 above)
Comment (yeargor) [reply src/main/java/Homework11/controller/HelloController.java:19]: I will fix it in a few moments

File: src/main/java/Homework11/model/User.java (hunk 2 above)
Comment (Alexmansar) [reply src/main/java/Homework11/model/User.java:15]: ok) o problem)) its good that you invest the question)

**Review by Alexmansar — APPROVED** [review]
@yeargor Please, merge it)) good job

**Pull request merged**